micro-deps/micro-deps-jax-rs/src/main/java/com/ofg/infrastructure/jaxrs/JaxRsServiceResolver.java

```diff
@@ -0,0 +1,17 @@
+package com.ofg.infrastructure.jaxrs;
+
+import com.google.common.base.Supplier;
+
+/**
+ * Allows to obtain JAX-RS proxy service interface implementation from its interface
+ */
+public interface JaxRsServiceResolver {
```

Comment:
This class needs to return an implementation of the interface which can handle cases when service in not discoverable yet:
Use case:
1. Service is Off
2. Get impl but calls to the it will fail
3. Service is On
3. Calls works